examples/http-basic-rest-service/http_basic_rest_service.md

```diff
@@ -1,6 +1,6 @@
 # REST service - Basic
 
-Ballerina supports writing RESTful with its first class service and resource concepts. The `accessor-name`, `service/resource name`, `data binding`, `path` and `query` parameter support helps to write meaningful APIs. The sample depicts the way of writing `GET` and `POST` resources.
+Ballerina language has first-class abstractions for service and resource concepts in the form of `service` and `resource functions`. A resource function consists of an accessor and path. A service can have a collection of resource functions. These abstractions allow mapping REST concepts such as operations, resource paths and resource representations cleanly into your program. `http:Service` can be used to write RESTful services. A service is defined with a base path, the path common to all resource paths. Each resource function is defined with the required operation such as `get`, `put`, `post`, etc and the path. Similar to regular functions resource functions have input parameters and return types that are mapped to the HTTP request and response.
```

Comment:
Ballerina language has first-class abstractions for service and resource concepts in the form of `service` and `resource functions`. A resource function consists of an accessor and path. A service can have a collection of resource functions. These abstractions allow mapping REST concepts such as operations, resource paths, and resource representations cleanly into your program. An `http:Service` can be used to write RESTful services. A service is defined with a base path, which is the path common to all resource paths. Each resource function is defined with the required operation such as `get`, `put`, `post`, etc., and the path. Similar to regular functions, resource functions have input parameters and return types that are mapped to the HTTP request and response.